app/src/main/res/layout/content_main.xml

```diff
@@ -56,18 +61,27 @@
         android:id="@+id/txtActivity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginTop="92dp"
         android:ems="10"
+        android:hint="@string/activity_hint"

```

Comment:
Added text hint and also created string in strings.xml and replaced text with @string. This removed this warning in android studio about using string values.

Reply:
Hints are a great utility.  A hint is essentially a label inside of an EditText (or similar component), which cuts down on the clutter on the screen.   Also less work for the developer, especially when using constraint layout!